src/main/src/test/java/org/geoserver/catalog/ResourcePoolTest.java

```diff
@@ -63,6 +64,8 @@
 import org.geotools.feature.collection.DecoratingFeatureCollection;
 import org.geotools.feature.collection.SortedSimpleFeatureCollection;
 import org.geotools.feature.simple.SimpleFeatureTypeBuilder;
+import org.geotools.gce.geotiff.GeoTiffFormat;
+import org.geotools.image.util.ImageUtilities;
```

Comment:
ImageUtilities is not being used and it's also resulting into a building problem.
/home/travis/build/geoserver/geoserver/src/main/src/test/java/org/geoserver/catalog/ResourcePoolTest.java:[68,30] error: package org.geotools.image.util does not exist